pkg/admission/conf/am_conf.go

```diff
@@ -351,8 +351,8 @@ func parseConfigRegexps(config map[string]string, key string, defaultValue strin
 	value := parseConfigString(config, key, defaultValue)
 	result, err := parseRegexes(value)
 	if err != nil {
-		log.Logger().Error(fmt.Sprintf("Unable to parse regex values '%s' for configuration '%s', using default value '%s'",
-			value, key, defaultValue), zap.Error(err))
+		log.Logger().Sugar().With(zap.Error(err)).Errorf("Unable to parse regex values '%s' for configuration '%s', using default value '%s'",
```

Comment:
don't use sugar, it's avoided in the project, do this instead:
```
log.Logger.Error("Unable to parse regex values, using default", zap.String("key", key), zap.String("value", value), zap.String("default", default))
```

Reply:
Updated it. I still keep `zap.Error(err)`. May I know why we don't use sugar? Is it cause of performance? Thank you.

Reply:
`zap.Error()` is fine. I remember inside Yunikorn, we avoid it for performance reasons. It probably wouldn't make a difference inside the Admission Controller, but at least be consistent with the existing code.